sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -1,14 +1,32 @@
 # Release History
 
-## 3.2.1 (Unreleased)
-
-### Features Added
+## 4.0.0-beta.1 (2021-10-05)
 
 ### Breaking Changes
 
-### Bugs Fixed
-
-### Other Changes
+- This new major version beta introduces a full redesign of the Azure Form Recognizer client library. To leverage features of the newest Form Recognizer service API (version "2021-09-30-preview" and newer), the new SDK is required, and application code must be changed to use the new clients. Please see the [Migration Guide](https://github.com/azure/azure-sdk-for-js/blob/main/sdk/formrecognizer/ai-form-recognizer/MIGRATION-v3_v4.md) for detailed instructions on how to update application code from version 3.x of the Form Recognizer SDK to the new version (4.x). Additionally, the [CHANGELOG](https://github.com/azure/azure-sdk-for-js/blob/main/sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md) for more detailed information.
+- This package targets Azure Form Recognizer service API version `2021-09-30-preview` and newer. It _is not_ compatible with the older Form Recognizer service API versions (2.0 and 2.1). To continue to use Form Recognizer API version 2.1, please use major version 3 of the client package (`@azure/ai-form-recognizer@^3.2.0`).
+- `FormRecognizerClient` has been replaced by `DocumentAnalysisClient`.
+  - The new `beginExtractLayout` method replaces the previous `beginRecognizeContent` method and its `-FromUrl` counterpart. Rather than an array of pages, the new method produces an object that has properties for `pages`, `tables`, and `styles`.
```

Comment:
instead of saying an array of pages, will it be better to say the specific type? so it is easier for the user to do the mapping?

Reply:
Good point, thanks!